src/cr/cube/matrix/cubemeasure.py

```diff
@@ -1001,8 +1134,12 @@ def _sliced_counts(cls, cube, slice_idx):
         need to extract only the selected counts, since we're "just" dealing with the
         tabulation.
         """
-        i = cls._regular_matrix_counts_slice(cube, slice_idx)
-        return (cube.counts[i], cube.unweighted_counts[i], cube.counts_with_missings[i])
+        i = cls._regular_matrix_slice(cube, slice_idx)
+        return (
+            cube.counts[i],
+            cube.unweighted_counts[i],
+            cube.counts_with_missings[i],
+        )
```

Comment:
I find `i` is misleading here, making one think it is an `int` as it commonly would be. Maybe `slice_expr = cls._cube_slice_expression(cube, slice_idx)` instead.